tests/test_packages/test_skills/test_gym.py

```diff
@@ -115,51 +66,14 @@ def test_gym(self, pytestconfig):
         shutil.copyfile(file_src, file_dst)
 
         # change number of training steps
-        skill_config_path = Path(
-            self.t, self.agent_name, "vendor", "fetchai", "skills", "gym", "skill.yaml"
+        setting_path = "vendor.{}.skills.gym.handlers.gym.args.nb_steps".format(
```

Comment:
use the standard cli config set approach rather than force set here

Reply:
Done.